snap/info.go

```diff
@@ -155,9 +155,16 @@ func MountDir(name string, revision Revision) string {
 	return filepath.Join(BaseDir(name), revision.String())
 }
 
-// MountFile returns the path where the snap file that is mounted is installed.
+// MountFile returns the path where the snap file that is mounted is installed,
+// using the default blob directory (dirs.SnapBlobDir).
 func MountFile(name string, revision Revision) string {
-	return filepath.Join(dirs.SnapBlobDir, fmt.Sprintf("%s_%s.snap", name, revision))
+	return MountFileInDir(dirs.SnapBlobDir, name, revision)
+}
+
+// MountFileInDir returns the path where the snap file that is mounted is
+// installed in a given root directory.
+func MountFileInDir(root, name string, revision Revision) string {
```

Comment:
root seems the wrong concept here, is just a directrory for the file, so  just `dir? ? same for the comment